collect_app/src/main/java/org/odk/collect/android/utilities/ApplicationConstants.java

```diff
@@ -88,4 +88,9 @@ public abstract static class Namespaces {
         public static final String XML_OPENROSA_NAMESPACE = "http://openrosa.org/xforms";
         public static final String XML_OPENDATAKIT_NAMESPACE = "http://www.opendatakit.org/xforms";
     }
+
+    public abstract static class AppStateKeys {
+
+        public static final String EDITED_FINALIZED_FORM = "editedFinalizedForm";
```

Comment:
One thing to consider: maybe this should be placed in `AppState` class. I'm not sure if keeping constant used in different parts of the app and different functionalities in one class is still a good idea.

Reply:
Yeah I'm also not sure about `ApplicationConstants`, but it did seem like the right place for this at the moment. Maybe we should look at splitting it up next time we want to add something?

I don't think that keys for `AppState` should live in the `AppState` class as the keys should be owned by the component/module/package **using** `AppState`. I guess you can think about it like a `Map` - it shouldn't need to know anything about the things it stores to do it's job of just storing them.